### ISSUE TITLE
Cases/Controls query code redundant in getPatientData.R?

### ISSUE THREAD

**Issue by turambar** [jmbanda/OHDSI-Phenotyping#3]
If I understand things correctly, lines 94-267 and 271-419 differ only in the patient subset to which they are being applied (cases vs. controls). Perhaps merge (put all patients into one list and iterate over that, and create a separate list with case/control labels) or turn into function. Since you're appending the features anyway, may as well just combine, I think...?

I'm pretty sure you're already doing this, but I thought I'd add it just in case. :)

**Comment by jmbanda** [comment]
Indeed. I am greatly reducing this part. I was being overly verbose in this
section and now I can easily collapse it into very generic functions.  Keep
the comments coming as they are very useful when I go over the full
conversion once I think it is ready for the public.
